tests/FakeItEasy.Specs/CancellationSpecs.cs

```diff
@@ -0,0 +1,240 @@
+﻿namespace FakeItEasy.Specs
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using FakeItEasy.Tests.TestHelpers;
+    using FluentAssertions;
+    using Xbehave;
+    using Xunit;
+
+    public class CancellationSpecs
+    {
+        public interface IFoo
+        {
+            int Bar();
+
+            int Bar(CancellationToken cancellationToken);
+
+            int Bar(object obj);
+
+            Task<int> BarAsync(CancellationToken cancellationToken);
+        }
+
+        [Scenario]
+        public void NoCancellationToken(IFoo fake, int result)
+        {
+            "Given a fake"
+                .x(() => fake = A.Fake<IFoo>());
+
+            "When a method which doesn't accept a cancellation token is called"
+                .x(() => result = fake.Bar());
+
+            "Then it doesn't throw and returns the default value"
+                .x(() => result.Should().Be(0));
+        }

```

Comment:
Ah, yes, I thought you were talking about another one. I included it as a sanity check, but you're right, it's covered elsewhere. I'll remove it.